esmvaltool/diag_scripts/perfmetrics/collect.ncl

```diff
@@ -204,12 +396,39 @@ begin
   ; Call plotting function
   if (diag_script_info@metric.eq."taylor") then
     plot = taylor_plot(wks, data_all, "grade")
+  else if (diag_script_info@metric.eq."SMPI") then
+    plot = circle_plot(wks, data_smpi, "performance_index", \
+                       data_smpi@ensemble_name)
   else
     plot = portrait_plot(wks, data_all, "grade")
   end if
+  end if
 
   log_info(" gv " + outfile_plot + "." + file_type)
 
   leave_msg(DIAG_SCRIPT, "")
 
+  ; collect meta-data and call ESMValMD function
+  nc_file = ncdf_outfile
+  plot_file = outfile_plot
+  domain = ("glob")
```

Comment:
```suggestion
  domains = ("global")
```

Reply:
Ok. I will keep that in mind.
The changes are ok with me.